buildstockbatch/cloud/docker_base.py

```diff
@@ -125,9 +125,9 @@ def _run_batch_prep(self, tmppath):
 
         This includes:
             - Weather files (:func:`_prep_weather_files_for_batch`)
```

Comment:
For my own understanding: previously this was generating the new samples locally but the assets were prepared before the new sampling in which case, it would have used the old sample. What if there was not any sample? how would that be handled ?

Reply:
> What if there was not any sample? how would that be handled ?

There would then not be a `buildstock.csv` in the assets.tar.gz, then Batch tasks would all each fail with "_FileNotFoundError: [Errno 2] No such file or directory: '/var/simdata/openstudio/lib/housing_characteristics/buildstock.csv'_".